src/Calendar/Calendar.js

```diff
@@ -465,21 +465,16 @@ export class Calendar extends Component {
                     }
                 );
 
-                // check if day is disabled
-                if (this.displayDisabled(day)) {
-                    days.push(<td className={dayClasses} key={copyDate}
-                        role='gridcell'><span className='fd-calendar__text'>{dateFormatted}</span>
-                    </td>);
-                } else {
-                    days.push(<td className={dayClasses} key={copyDate}
-                        onClick={() => this.dateClick(copyDate, enableRangeSelection)} role='gridcell'><span className='fd-calendar__text'>{dateFormatted}</span>
-                    </td>);
-                }
+                days.push(<td className={dayClasses} key={copyDate}
+                    onClick={!this.displayDisabled(day) ? () => this.dateClick(copyDate, enableRangeSelection) : null
+                    } role='gridcell' > <span className='fd-calendar__text'>{dateFormatted}</span>
+                </td >);
```

Comment:
I like this much better -- less chance that a change to calendar days get missed.

Minor nit:  from a code readability standpoint, I think I'd reformat and put a prop on each line like:
```
days.push(
    <td
        className={dayClasses}
        key={copyDate}
        onClick={!this.displayDisabled(day) ? () => this.dateClick(copyDate, enableRangeSelection) : null}
        role='gridcell'>
        <span className='fd-calendar__text'>{dateFormatted}</span>
    </td>
);
```

Reply:
👍 yes it does look better, done